.github/workflows/test.yml

```diff
@@ -17,6 +18,7 @@ jobs:
       matrix:
         include:
         - python-version: "3.8"
+          toxenv: rules
```

Comment:
No strong opinion, but we usually test “extras” with the latest Python version, rather than the lowest supported.

Reply:
Updated 👍